mali: Support building against 5.7

Linux 5.7 removed PTR_RET which is an alias of PTR_ERR_OR_ZERO since 3.12.

Signed-off-by: Jonathan Liu <net147@gmail.com>

--- a/src/devicedrv/mali/linux/mali_memory_dma_buf.c
+++ b/src/devicedrv/mali/linux/mali_memory_dma_buf.c
@@ -258,7 +258,11 @@ int mali_dma_buf_get_size(struct mali_session_data *session, _mali_uk_dma_buf_ge
 	buf = dma_buf_get(fd);
 	if (IS_ERR_OR_NULL(buf)) {
 		MALI_DEBUG_PRINT_ERROR(("Failed to get dma-buf from fd: %d\n", fd));
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 7, 0)
+		return PTR_ERR_OR_ZERO(buf);
+#else
 		return PTR_RET(buf);
+#endif
 	}
 
 	if (0 != put_user(buf->size, &user_arg->size)) {
